pkg/validating-webhooks/apischeme_validating_admission_hook.go

```diff
@@ -0,0 +1,119 @@
+package validatingwebhooks
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/openshift/cloud-ingress-operator/pkg/apis/cloudingress/v1alpha1"
+
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+// TODO REMOVE ME
+// Most of this is based on the examples in github.com/openshift/hive/pkg/validating-webhooks
+
+const (
+	// TODO I have no idea what these values should actually be.
```

Comment:
Fairly certain these values are correct, but was never able to get to fully validate them.